packages/kbn-test/src/jest/run.ts

```diff
@@ -73,7 +79,14 @@ export function runJest(configName = 'jest.config.js') {
     process.env.NODE_ENV = 'test';
   }
 
-  run();
+  run().then(() => {
+    // Success means that tests finished, doesn't mean they passed.
+    reportTime(runStartTime, 'total', {
+      success: true,
+      isXpack: cwd.includes('x-pack'),
+      testFiles: map(testFiles, (testFile) => relative(testFile, cwd)),
```

Comment:
This doesn't seem to be working for me:

node scripts/jest src/cli_encryption_keys/generate.test.js
```
{
  "cpuSpeed": 1500,
  "nestedTiming": false,
  "totalMem": 33432760320,
  "testFiles": [
    "../../.."
  ],
```

Reply:
@jbudz fixed!